src/components/context-test/ContextTest.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="va-test" :style="computedStyles">
+    <slot>Test</slot>
+  </div>
+</template>
+
+<script>
+import { ColorThemeMixin } from '../../services/ColorThemePlugin'
+import { ContextProviderPluginKey } from '../../services/ContextProvidePlugin'
+
+export default {
+  name: 'va-test',
+  mixins: [ColorThemeMixin],
+  inject: { _$context: [ContextProviderPluginKey] },
+  props: {
+    color: {
+      type: String,
+      default () {
+        return this._$context.getComponentContextProps(this._$context.config, this.$options.name, 'color')
```

Comment:
> this._$context.config
We can get rid of `this._$context.config` as it could be taken from object context.